apps/hello/templates/services/app/cronjob_hello.yml.j2

```diff
@@ -0,0 +1,33 @@
+apiVersion: batch/v1beta1
```

Comment:
in commit message:
s/currect/correct
s/the other one/the other ones
s/active only if the are switc in the current task/activated only when they are switched to the current stack